examples/erosion2/app.py

```diff
@@ -0,0 +1,83 @@
+import os
```

Comment:
I recommend putting a link to the tutorial in the docstring here.

Reply:
Done.